10a-walkthrough-6.Rmd

```diff
@@ -0,0 +1,677 @@
+
+```{r}
+model_data %>% 
+    group_by(year) %>% 
+    summarise(median_ratio = median(ratio))
+```
+
+Now let's visualize this using our boxplots: 
+
+```{r visualize female to male ratio}
+model_data %>% 
+    gather(gender, students, c(f, m)) %>% 
+    ggplot(aes(x = year, y = students, color = gender)) + 
+    geom_boxplot() + 
+    scale_y_continuous(labels = scales::comma) + 
+    labs(title = "Median male and female student counts in special education", 
+         subtitle = "Ages 6-21. Does not include outlying areas and freely associated states", 
+         x = "", 
+         y = "", 
+         caption = "Data: US Dept of Education")
+```
+
+```{r visualize female to male ratio 2}
+model_data %>% 
+    gather(gender, students, c(f, m)) %>%
+    ggplot(aes(x = year, y = students)) + 
+    geom_boxplot() + 
+    facet_grid(~ gender) +
+    scale_y_continuous(labels = scales::comma) + 
+    labs(title = "Median male and female student counts in special education", 
+         subtitle = "Ages 6-21. Does not include outlying areas and freely associated states", 
+         x = "", 
+         y = "", 
+         caption = "Data: US Dept of Education")
+```
+
+```{r visualize female to male ratio 3}
+library(ggridges)
+
+model_data %>% 
+    gather(gender, students, c(f, m)) %>%
+    ggplot(aes(x = students, y = gender)) + 
+    geom_density_ridges() +
+    scale_x_continuous(labels = scales::comma) +
+    labs(title = "Density estimates for male and female student counts in special education", 
+         subtitle = "Ages 6-21. Does not include outlying areas and freely associated states", 
+         x = "", 
+         y = "", 
+         caption = "Data: US Dept of Education")
+```
+
+Once we learned from our model that male to female ratios did not change in any meaningful way from 2012 to 2017 and that the median ratio across states was about 2 male students to every female student, we can present these two ideas using this plot. When discussing the plot, it helps to have your model output available to you so you can reference specific actual coefficient estimates when needed. 
+
+# 11.4 Aggregate Data as Context for Student Data 
+
+Education data science is about using data science tools to learn about and improve the lives of our students. So why choose a publicly available aggregate dataset to analyze instead of a student level dataset? We chose to build this analysis walkthrough using an aggregate dataset because it reflects an analysis that an education data scientist would typically do. Using student level data requires that the education data scientist is either an employee of the school agency that serves the students or that she works under a memorandum of understanding (MOU) that allows her to access this data. Without either of these conditions, the education data scientist learns about the student experience by working on publicly available datasets, almost all of which are aggregated student level datasets.
```

Comment:
thoughts on moving this to the beginning of the chapter? this could be a nice complement to the very first section - on publicly available data. open to keeping this here.